tests/flow/test_enhanced_coverage.py

```
@@ -0,0 +1,293 @@
+        
+        # Test setting valid config
+        result = self.db.execute_command("GRAPH.CONFIG", "SET", "CACHE_SIZE", "30")
+        assert result == "OK"
+        
+        # Test setting invalid config value
+        try:
+            self.db.execute_command("GRAPH.CONFIG", "SET", "THREAD_COUNT", "0")
+            # Should reject invalid values
+        except redis.ResponseError:
+            # Expected for invalid value
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(redis.ResponseError)` instead of `try`-`except`-`pass`

(SIM105)